LethalAPI.Core/Plugin.cs

```diff
@@ -10,32 +10,58 @@ namespace LethalAPI.Core;
 // ReSharper disable InconsistentNaming
 // ReSharper disable MemberCanBePrivate.Global
 #pragma warning disable SA1401 // field should be made private
+using System;
 
 using BepInEx;
 using BepInEx.Logging;
 using HarmonyLib;
+using MEC;
 
 /// <inheritdoc />
 [BepInPlugin(PluginInfo.PLUGIN_GUID, PluginInfo.PLUGIN_NAME, PluginInfo.PLUGIN_VERSION)]
 public class Plugin : BaseUnityPlugin
 {
+    /// <summary>
+    /// Gets the singleton for a plugin.
+    /// </summary>
+    public static Plugin Singleton = null!;
```

Comment:
This should probably be called "Instance"?

Reply:
I always have named them singletons. I suppose this is a good naming convention to keep with though.